typescript/cli/src/ism/read.ts

```diff
@@ -28,7 +33,8 @@ export async function readIsmConfig({
     config = await ismReader.deriveIsmConfig(address);
     stringConfig = stringifyObject(config, resolveFileFormat(out), 2);
   } else {
-    const provider = await context.altVmProvider.get(chain);
+    const provider = context.altVmProvider.get(chain);
+    assert(provider, `Cannot find ${provider}`);
     const ismReader = new AltVMIsmReader(
```

Comment:
Using the modules like is OK but reminds me that I forgot one missing piece of the puzzle. Look at the ModuleProvider<M> interface. There are implementation in deploy-sdk for all 4 current module types. It provides a more uniform alternative to new/create/new-reader. I forgot to add an implementation of this to deploy-sdk. We can add it in the next PR to keep this one focused.